packages/thirdweb/src/extensions/prebuilts/deploy-published.ts

```diff
@@ -335,13 +338,15 @@ async function directDeploy(options: {
   compilerMetadata: CompilerMetadata;
   contractParams?: Record<string, unknown>;
   salt?: string;
+  metadataUri?: string;
 }): Promise<string> {
   const { account, client, chain, compilerMetadata, contractParams, salt } =
     options;
 
   const { deployContract } = await import(
     "../../contract/deployment/deploy-with-abi.js"
   );
+  const isStylus = options.compilerMetadata.metadata.language === "rust";
```

Comment:
How come you're not looking at compiler type here?

Reply:
we have that in publish data, but not in compiler metadata (so, I add language as rust and check it later).